db/migrate/20220407130737_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.references :post, null: false, foreign_key: { to_table: :posts }
```

Comment:
- [ ] [OPTIONAL] in this case you don't need to specify the `to_table:` parameter. rails can figure out this relationship based on the reference name.

```suggestion
      t.references :post, null: false, foreign_key: true
```